test/scheduler_test.py

```diff
@@ -112,5 +112,129 @@ def __init__(self):
 
         worker.prune(TmpCfg())
 
+    @with_config({'scheduler': {'disable-num-failures': '44'}})
+    def test_scheduler_with_task_level_config(self):
+        cps = luigi.scheduler.CentralPlannerScheduler()
```

Comment:
Sorry, as I merged this you have to update this line :)

https://github.com/spotify/luigi/pull/1781